### PR TITLE
[HttpClient-Vertx] Live test for vertex http client 

### PR DESCRIPTION
**Pull request by mssfang**
Live testing Vertx HttpClient in 

[X] AppConfiguration SDK
[X] Container Registry SDK
[X] Text Analytics SDK

### CODE REVIEW

File: sdk/appconfiguration/tests.yml [hunk 1]
```diff
@@ -14,3 +14,8 @@ stages:
         AZURE_CLIENT_ID: $(APPCONFIGURATION_CLIENT_ID)
         AZURE_CLIENT_SECRET: $(APPCONFIGURATION_CLIENT_SECRET)
         AZURE_TENANT_ID: $(APPCONFIGURATION_TENANT_ID)
+      MatrixConfigs:
+        - Name: App_Configuration_live_test
+          Path: sdk/appconfiguration/platform-matrix.json
+          Selection: sparse
+          GenerateVMJobs: true
```
Comment (joshfree) [inline sdk/appconfiguration/tests.yml:21]: nit: missing new line

File: sdk/textanalytics/platform-matrix-ci.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "matrix": {
+    "$IMPORT": "eng/pipelines/templates/stages/platform-matrix.json"
+  },
+  "include": [
+    {
+      "Agent": { "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" } },
```
Comment (joshfree) [inline sdk/textanalytics/platform-matrix-ci.json:7]: should this be on 22.04 at this point?

**Review by joshfree — APPROVED** [review]

File: sdk/textanalytics/platform-matrix-ci.json (hunk 2 above)
Comment (mssfang) [reply sdk/textanalytics/platform-matrix-ci.json:7]: We are still using the 20.04 in our repo. Will need to create another PR to upgrade all if needed.

**Review by joshfree — APPROVED** [review]

**Review by alzimmermsft — APPROVED** [review]

**Pull request merged**